hondana/http.py

```diff
@@ -292,7 +292,8 @@ async def _get_token(self) -> str:
         self.__refresh_token = refresh_token
         return token
 
-    def _get_expiry(self, token: str) -> datetime.datetime:
+    @staticmethod
+    def _get_expiry(token: str) -> datetime.datetime:
```

Comment:
This seems to be a jetbrains linter-ism that I don't particularly like. I don't suppose you know why this is beneficial?
Otherwise I'd opt for reverting this.

Reply:
Yeh this is a JetBrains thing. 

Im not too sure but i think the main benefits are less memory, although negligible, due to not using bound methods. I think they are also able to be called without an instance of the class which is useful.

Reply:
Yeah it's probably nit-picky but I'd prefer it stay a bound method.

I'll do some more looking into the benefits of this, as I do like micro-optimizations.

Reply:
Sounds good to me.